YoloDotNet/Data/YoloBase.cs

```diff
@@ -15,6 +18,10 @@ public abstract class YoloBase : IDisposable
         protected readonly ParallelOptions _parallelOptions;
         private readonly bool _useCuda;
 
+        private int tensorBufferSize;
+        private ArrayPool<float> customSizeFloatPool;
+        protected ArrayPool<ObjectResult> customSizeObjectResultPool;
```

Comment:
Prefix private variables with an underscore _ to keep things consistent with the rest of the code